hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/compact/strategy/DayBasedCompactionStrategy.java

```diff
@@ -60,6 +62,17 @@ public Comparator<String> getComparator() {
     return comparator;
   }
 
+  @Override
+  public List<HoodieCompactionOperation> orderAndFilter(HoodieWriteConfig writeConfig,
```

Comment:
should we extend from `BoundedIOCompactionStrategy` and by default does not limit the IO boundary(make it a maximum default value maybe)?

Reply:
Maybe add a boolean value is more reasonable. `BoundedIOCompactionStrategy` has a default config `hoodie.compaction.target.io` with value 512G.  Following your idea，we should add a similar new config like `hoodie.compaction.target.io` which make it maximum default value. That seems a little conflict.

Reply:
`512G` itself is a value big enough for most of the production use cases, we can use it IMO.